.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           name: Test app
           working_directory: hello-world/app
           command: |
-            oasis-chain &
+            oasis chain &
```

Comment:
you don't need this because you're pulling the tutorial down from git during build